editor/src/clj/editor/render_pb.clj

```diff
@@ -27,40 +27,40 @@
   (:import
    [com.dynamo.render.proto Render$RenderPrototypeDesc]))
 
-(g/defnode NamedMaterial
+(g/defnode NamedRenderResource
   (property name g/Str
             (dynamic visible (g/constantly false)))
-  (property material resource/Resource
-            (value (gu/passthrough material-resource))
+  (property render-resource resource/Resource
+            (value (gu/passthrough resource))
             (set (fn [evaluation-context self old-value new-value]
                    (let [project (project/get-project (:basis evaluation-context) self)
-                         connections [[:resource :material-resource]
+                         connections [[:resource :resource]
                                       [:build-targets :dep-build-targets]]]
                      (concat
                        (if old-value
                          (project/disconnect-resource-node evaluation-context project old-value self connections)
-                         (g/disconnect project :nil-resource self :material-resource))
+                         (g/disconnect project :nil-resource self :resource))
                        (if new-value
                          (:tx-data (project/connect-resource-node evaluation-context project new-value self connections))
-                         (g/connect project :nil-resource self :material-resource))))))
-           (dynamic visible (g/constantly false)))
+                         (g/connect project :nil-resource self :resource))))))
+            (dynamic visible (g/constantly false)))
 
-  (input material-resource resource/Resource)
+  (input resource resource/Resource)
   (input dep-build-targets g/Any)
 
-  (output named-material g/Any (g/fnk [_node-id name material]
+  (output dep-build-targets g/Any (gu/passthrough dep-build-targets))
+  (output named-render-resource g/Any (g/fnk [_node-id name render-resource]
                                  {:name name
-                                  :material material}))
-  (output dep-build-targets g/Any (gu/passthrough dep-build-targets)))
+                                  :path render-resource})))
 
-(defn- make-named-material-node
-  [graph-id render-node name material-resource]
+(defn- make-named-render-resource-node
+  [graph-id render-node name render-resource-path]
```

Comment:
We use `path` only to refer to string paths. This should be a `Resource`. So I guess that makes this a `render-resource-resource` 😅.